src/main/java/io/temporal/samples/dsl/models/Parrallel.java

```diff
@@ -0,0 +1,49 @@
+package io.temporal.samples.dsl.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.temporal.workflow.Async;
+import io.temporal.workflow.CancellationScope;
+import io.temporal.workflow.Promise;
+import io.temporal.workflow.Workflow;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class Parrallel {
+  public Statement branches[];
+
+  @JsonCreator
+  public Parrallel(@JsonProperty("branches") Statement branches[]) {
+    this.branches = branches;
+  }
+
+  public void execute(Map<String, String> bindings) {
+    // You can use the context passed in to activity as a way to cancel the activity like standard
+    // GO way.
+    // Cancelling a parent context will cancel all the derived contexts as well.
```

Comment:
This comment doesn't make any sense to a Java developer that never learned Go.

Reply:
removed